users/views.py

```diff
@@ -67,4 +68,11 @@ def post(self, request):
             return JsonResponse({'message':'LOGIN_SUCCESS','token':access_token,'user_name':user.nickname}, status=200)
 
         except KeyError:
-            return JsonResponse({'message':'KEY_ERROR'}, status=400)
\ No newline at end of file
+            return JsonResponse({'message':'KEY_ERROR'}, status=400)
+
+class DeleteAccount(View):
```

Comment:
```suggestion
class Account(View):
```
계정정보 조회, 수정 , 삭제 등 범용적으로 만들 수 있을 것 같아요